.pre-commit-config.yaml

```
@@ -2,4 +2,27 @@ repos:
 -   repo: https://github.com/psf/black
     rev: 19.10b0
```

Black is now in version `20.8b1` 😉 
```suggestion
    rev: 20.8b1
```